src/dreyfus/src/dreyfus_rpc.erl

```diff
@@ -80,8 +80,13 @@ info_int(DbName, DDoc, IndexName) ->
         {ok, Index} ->
             case dreyfus_index_manager:get_index(DbName, Index) of
                 {ok, Pid} ->
-                    Result = dreyfus_index:info(Pid),
-                    rexi:reply(Result);
+                    case dreyfus_index:info(Pid) of
+                        {ok, Fields} ->
+                            Info = [{signature, Index#index.sig} | Fields],
+                            rexi:reply({ok, Info});
+                        {error, Reason} ->
```

Comment:
We don't expect any other return values besides `{ok, Fields}` or `{error, Reason}`?

Saw that the `handle_message` handler the replies expected an `EXIT` message (https://github.com/apache/couchdb/pull/3927/files#diff-6eca3eab5f219bc57196b2b0cdbe603069f36ec212eab6a163d541261fd2f91bR78)

```
handle_message({'EXIT', _}, Worker, {Counters, Acc}) ->`
   ...
```

Which is a bit unusual, because it's the only worker message handler which handles it, but I didn't know if it was an actual worker process exit, or the `EXIT` atom passed through somehow from clouseau which would now fail as clause match.

Reply:
actually Clouseau either returns {ok, Info} or fails (and we'd get a link exit). I'll make that explicit on the dreyfus side